### PR TITLE
Update to PHP 8.2 and Symfony 6.4

### PR DESCRIPTION
**Pull request by pmeulen**
Includes tiqr-server-libphp 4.1.0 which updates from the old FCM API to the Google FCM HTTP v1 API
Includes CI changes

### CODE REVIEW

File: config/openconext/parameters.yaml.dist [hunk 1]
```diff
@@ -58,7 +58,8 @@ parameters:
                 certificate: 'absolute path to certificate'
                 environment: production
             firebase:
-                apikey: 'Your Firebase API KEY'
+                projectId: 'FCM project ID'
+                credentialsFile: 'absolute path to file with service account key'
```
Comment (phavekes) [inline config/openconext/parameters.yaml.dist:62]: I'm missing the `cacheTokens` parameter here

**Review by phavekes — CHANGES_REQUESTED** [review]

**Review by phavekes — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -58,11 +58,11 @@
     "slevomat/coding-standard": "^8.13",
     "soyuka/contexts": "^3.3",
     "squizlabs/php_codesniffer": "^3.7",
-    "symfony/browser-kit": "6.4.*",
+    "symfony/browser-kit": "^6.4",
```
Comment (MKodde) [inline composer.json:61]: It is actually slightly better to pin to a `6.4.*` version instead of allowing any version of 6.4 and higher but lower than 5 (`^6.4`). That way we can never install a 6.5 version which would probably need maintenance to the app to work right.

As we know there will never be a 6.5 version we can leave it like this. But normally I would prefer to revert this change.

File: composer.json [hunk 3]
```diff
@@ -169,7 +173,8 @@
       "/src/Features",
       ".gitignore",
       ".travis.*",
-      "ci/qa/behat.yml"
+      "ci/qa/behat.yml",
+      "/vendor/google/apiclient-services/src/*"
```
Comment (MKodde) [inline composer.json:177]: Are you intentionally excluding this folder? I would think that code is important to make it work.

**Review by MKodde — COMMENTED** [review]
Nice work on this PR! See two comments from my critical mind.

File: composer.json (hunk 3 above)
Comment (pmeulen) [reply composer.json:177]: Yes, this is intentional. /vendor/google/apiclient-services/src/* contains automatically generated PHP classes for ALL the Google APIs. We do not use these. The classes required for the apiclient are not removed, these are in different directories.
There two issues with this:
1. It is huge
2. It contains filenames that are so long that it breaks the MAX_PATH length, which makes the tar step file.

File: composer.json (hunk 2 above)
Comment (pmeulen) [reply composer.json]: Thx, fixed.
